### PR TITLE
use packaging to check parsl versions

### PR DESCRIPTION
**Pull request by lgray**

### CODE REVIEW

File: tests/test_parsl.py [hunk 1]
```diff
@@ -4,8 +4,9 @@
 import multiprocessing
 import sys
 import pytest
+from packaging import version
 
-if sys.version_info.major == 3 and sys.version_info.minor >= 8 and sys.platform.startswith("darwin"):
+if version.parse(sys.version.split()[0]) >= version.parse('3.8') and sys.platform.startswith("darwin"):
```
Comment (nsmith-) [inline tests/test_parsl.py:9]: What's up with the other method?

Comment (lgray) [reply tests/test_parsl.py:9]: it's still running parsl on darwin?

Comment (lgray) [reply tests/test_parsl.py:9]: checking if it's this or the 'startswith'

Comment (lgray) [reply tests/test_parsl.py:9]: oh - that's weird... the CI that failed waiting for parsl to run now passes and doesn't run it... what....

Comment (lgray) [reply tests/test_parsl.py:9]: however there is a ubuntu that stalled on parsl for py 3.8.... hmm.

**Comment by lgray** [comment]
not needed.

**Pull request closed without merging**